src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+// import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+ 
+public class Intake extends SubsystemBase{
+    private final DigitalInput input0 = new DigitalInput(0);
+    private final DigitalInput input1 = new DigitalInput(1);
+    private final DigitalInput input2 = new DigitalInput(2);
+
+    private final TalonFX intakeMotor = new TalonFX(0);
+    /** Creates a wait function that can be called on using wait() */
+    public static void wait(int ms) {
+        try
+        {
+            //Uses milliseconds as sleep time
+            Thread.sleep(ms);
+        }
+        catch(InterruptedException ex){
+            Thread.currentThread().interrupt();
+        }
+    }
+    boolean inside = false;
+    boolean readyToShoot = false;
+    boolean failSafe = false;
+    boolean status = intakeMotor.isAlive();
+    double intakeMotorSpeed = 0;
+    /** Creates a new intake. */
```

Comment:
FWIW, comments like this aren't super necessary. If you only have one constructor and it's the default one, you're not really adding much by just saying that you're creating an instance of the class here. A good use of the JavaDoc comment here would be if you had multiple constructors and they differed in some way that needed to be called out. The comments answer "why" and only answer "what" if the code is confusing. And even then I tend to try to clean up the code first.